### PR TITLE
fix: adjust dns build design

### PR DESCRIPTION
**Pull request by lucstjean-ssc**
Removing the modification of the SOA and NS records for the core and sub DNS zones as it is not advised as described [here.](https://cloud.google.com/dns/docs/zones#create-pub-zone)

A comment will be placed in the client-landing-zone package setters to instruct the operator to set the NS record's nameservers of the client subzone hosted in the core DNS zone to the same value as the subzone nameservers.

### CODE REVIEW

**Review by alaincormier-ssc — APPROVED** [review]

**Review by seanwilkens-ssc — APPROVED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM

**Review by cartyc — APPROVED** [review]

File: solutions/client-landing-zone/setters.yaml [hunk 1]
```diff
@@ -125,10 +125,9 @@ data:
   #
   dns-project-id: dns-project-12345
   dns-name: "client-name.example.com."
-  # Select a primary nameserver for the SOA record
-  # The nameservers must be part of the same group of nameservers (a,b,c,d) as the primary nameserver
-  dns-primary-nameserver: |
-    - "ns-cloud-a1.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 259200 300"
+  # The nameservers below must be updated to match the same nameservers as the client DNS subzone
```
Comment (davelanglois-ssc) [inline solutions/client-landing-zone/setters.yaml:128]: yeah, that means that this package will have to be deployed twice. once to create the client public dns zone, then you capture the nameservers that have been assigned by google and you update this setter and you deploy again. @lucstjean-ssc can you create a pbi to update the client onboading procedure accordingly.

Comment (lucstjean-ssc) [reply solutions/client-landing-zone/setters.yaml:128]: @davelanglois-ssc - Sure, no problem.

Comment (davelanglois-ssc) [reply solutions/client-landing-zone/setters.yaml:128]: a good example where I think kpt apply-time-mutation could have been used

Comment (davelanglois-ssc) [reply solutions/client-landing-zone/setters.yaml:128]: you can resolve this conversation when you are ready to merge

Comment (lucstjean-ssc) [reply solutions/client-landing-zone/setters.yaml:128]: Thanks @davelanglois-ssc

Comment (davelanglois-ssc) [reply solutions/client-landing-zone/setters.yaml:128]: @cartyc see above comment related to apply-time-mutation

**Pull request merged**